### PR TITLE
754 add duolingo english test score to mini intake language assessment section

### PR DESCRIPTION
**Pull request by MuhannedNoman**

### CODE REVIEW

**Review by samschlicht — DISMISSED** [review]
Nice work, @MuhannedNoman! No suggestions from me, it all looks good to go. 👍

File: ui/admin-portal/src/app/components/util/det-score-validation/det-score-validation.component.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  constructor() { }
+
+  ngOnInit(): void {
+    this.value = this.control.value;
+    this.regex = new RegExp('^(1[0-9]|[2-9][0-9]|1[0-5][0-9]|160)$');
+  }
+
+  update() {
+    if (this.value !== null) {
+      if (!this.regex.test(this.value)) {
```
Comment (sadatmalik) [inline ui/admin-portal/src/app/components/util/det-score-validation/det-score-validation.component.ts]: This of course works but personally I'd just test that the value is in the required range i.e. >= 10 and <= 160 for readability

**Review by sadatmalik — DISMISSED** [review]
Nicely done!

**Review by sadatmalik — DISMISSED** [review]

**Review by sadatmalik — APPROVED** [review]

**Review by sadatmalik — DISMISSED** [review]

**Review by sadatmalik — APPROVED** [review]
Approved

**Review by sadatmalik — DISMISSED** [review]

**Review by sadatmalik — DISMISSED** [review]

**Review by sadatmalik — DISMISSED** [review]

**Review by sadatmalik — APPROVED** [review]

**Pull request merged**